### ISSUE TITLE
Link defence config inputs to labels

### ISSUE THREAD

**Issue by pmarsh-scottlogic** [gsproston-scottlogic/prompt-injection#536]
Defence configuration inputs would ideally have linked label elements. E.g. `max message length` should become a `<label>` with the `for` property set to the id of the input. [example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). Remove any `aria-label`s from the inputs. ([more information](https://dequeuniversity.com/rules/axe/4.8/label?application=AxeChrome))

![Image](https://github.com/ScottLogic/prompt-injection/assets/118171430/8e8f812d-719e-401b-9563-1fd732bfc8e3)

Similarly we should link the toggle defence checkbox inputs to the defence titles if we can.

Things are working fine at the moment, it's just not best practise. Therefore this is low priority
